src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -693,7 +693,7 @@ public bool IsStale(DocumentsOperationContext databaseContext, long? cutoff = nu
 
         public enum IndexProgressStatus
         {
-            Faulty = -1,
+            BadIndex = -1,
```

Comment:
Not sure if 'bad' is a good idea. We already have well defined index states: faulty and error. Maybe let's leave `Faulty` and introduce `Error ` here?